scripts/run-mackoon-tests

```diff
@@ -106,7 +106,7 @@ done
 # Remove exit trap, so we can propagate exit value from pytest
 trap - EXIT
 
-# Run integration tests
-pytest -m integration
+# Run mackoon tests
+pytest -m mackoon
```

Comment:
We should probably use `-rap` here too, for consistent output.

Reply:
Fixed. Also changed all mackoon to mockoon filenames and strings in source files.